src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -342,18 +351,16 @@ def forward(
         query = query.permute(0, 2, 1, 3)
         # value: batch_size x num_attention_heads x seq_length x head_dim
 
+        # Note that this cast is quite ugly, but is not implemented before ROPE as the original codebase keeps the key in float32 all along the computation.
+        # Reference: https://github.com/kingoflolz/mesh-transformer-jax/blob/f8315e3003033b23f21d78361b288953064e0e76/mesh_transformer/layers.py#L128
```

Comment:
not sure we need this anhymore

Reply:
yeah, seems like it was not needed as we cast it back to fp32 just after a few lines